packages/runner/src/suite.ts

```diff
@@ -325,7 +347,11 @@ function createSuiteCollector(
           : options.todo
             ? 'todo'
             : 'run',
-      meta: options.meta ?? Object.create(null),
+      meta: {
+        ...Object.create(null),
```

Comment:
What is the purpose of this line? If meta should be kept with a null prototype, you need to create it first and then assign properties with `Object.assign`, for example. This line just does nothing